src/index.js

```diff
@@ -45,11 +43,19 @@ class MCServer extends EventEmitter {
   }
 
   connect (options) {
-    const version = require('minecraft-data')(options.version).version
-    if (!supportedVersions.some(v => v.includes(version.majorVersion))) {
-      throw new Error(`Version ${version.minecraftVersion} is not supported.`)
+    const mcData = require('minecraft-data')(options.version)
```

Comment:
no need to use both prismarine-registry and mcdata

prismarine-retry is a replacement of mcdata

you can use only prismarine-registry